recipe/doc/recipe_guide.md

```diff
@@ -16,7 +16,8 @@ Of particular interest from a conda packaging perspective is the binary backward
 This guide assumes that you already know how to write and compile code that supports such behavior.
 If your library is properly configured as such, you will need to do a bit of extra work to ensure that your conda package supports this as well:
 
-- You must [ignore the `run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) of any CUDA packages that your package depends on. Otherwise, the `run_exports` would require the runtime libraries to have versions equal to or greater than the versions used to build the package.
+- You must [ignore_run_exports_from`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) of any CUDA library packages that your package depends on. Otherwise, the `run_exports` would require the runtime libraries to have versions equal to or greater than the versions used to build the package.
```

Comment:
Ignoring run exports is just needed for `-dev` packages, right?
```suggestion
- You must define [`ignore_run_exports_from`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) for any CUDA library `-dev` packages that your package depends on. Otherwise, the `run_exports` would require the runtime libraries to have versions equal to or greater than the versions used to build the package.
```